imagepicker/src/main/java/com/esafirm/imagepicker/helper/LocaleManager.java

```diff
@@ -36,4 +37,30 @@ public static Context updateResources(Context context) {
         }
         return context;
     }
+
+    private static Locale zhCheck(Locale newLocaleLanguage) {
+        final String ZH = "zh";
+        final String TW = "TW";
+        final String CN = "CN";
+
+        Locale mLocale;
```

Comment:
Hi @stu01509 
Please remove the hungarian notation from the variable name

`mLocale` -> `locale`
`mNewLocaleLanguage` -> `newLocaleLanguage`